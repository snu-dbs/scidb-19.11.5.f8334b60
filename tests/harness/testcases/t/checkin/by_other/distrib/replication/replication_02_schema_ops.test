--setup
--start-query-logging


--test

########
# create
########

create array DAY_HA <daystr: string>[day=0:6,1,0] distribution hashed
show(DAY_HA)

create array DAY_RE <daystr: string>[day=0:6,1,0] distribution replicated
# show dist
show(DAY_RE)


########
# input
########

--stop-query-logging
--echo "show('input(DAY_HA, <DATAFILE>)','afl')"
show('input(DAY_HA, \'${TEST_DATA_DIR}/doc/m4x4_missing.txt\')', 'afl')

# show dist
--echo "show('input(DAY_RE, <DATAFILE>)','afl')"
show('input(DAY_RE, \'${TEST_DATA_DIR}/doc/m4x4_missing.txt\')', 'afl')
--start-query-logging

########
# build
########
# TODO: test needs upgrade to show output distribution == default distribution
# TODO: this would replace _show2_nodist
_show2_nodist('build(DAY_HA, \'foo\')','afl')
_show2_nodist('build(DAY_RE, \'foo\')','afl')

--cleanup
remove(DAY_HA)
remove(DAY_RE)
