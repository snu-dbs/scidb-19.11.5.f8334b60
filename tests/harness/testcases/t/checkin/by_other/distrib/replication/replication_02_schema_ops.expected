SCIDB QUERY : <create array DAY_HA <daystr: string>[day=0:6,1,0] distribution hashed>
Query was executed successfully

SCIDB QUERY : <show(DAY_HA)>
{i} schema,distribution,etcomp
{0} 'DAY_HA<daystr:string> [day=0:6:0:1]','hashed','none'

SCIDB QUERY : <create array DAY_RE <daystr: string>[day=0:6,1,0] distribution replicated>
Query was executed successfully

SCIDB QUERY : <show(DAY_RE)>
{i} schema,distribution,etcomp
{0} 'DAY_RE<daystr:string> [day=0:6:0:1]','replicated','none'

"show('input(DAY_HA, <DATAFILE>)','afl')"
{i} schema,distribution,etcomp
{0} 'DAY_HA<daystr:string> [day=0:6:0:1]',null,null

"show('input(DAY_RE, <DATAFILE>)','afl')"
{i} schema,distribution,etcomp
{0} 'DAY_RE<daystr:string> [day=0:6:0:1]',null,null

SCIDB QUERY : <_show2_nodist('build(DAY_HA, \'foo\')','afl')>
{i} schema
{0} 'DAY_HA<daystr:string> [day=0:6:0:1]'

SCIDB QUERY : <_show2_nodist('build(DAY_RE, \'foo\')','afl')>
{i} schema
{0} 'DAY_RE<daystr:string> [day=0:6:0:1]'

SCIDB QUERY : <remove(DAY_HA)>
Query was executed successfully

SCIDB QUERY : <remove(DAY_RE)>
Query was executed successfully

